app/assets/stylesheets/login.scss

```diff
@@ -0,0 +1,62 @@
+@media (min-width: 768px) {
+  .omb_row-sm-offset-3 div:first-child[class*="col-"] {
+    margin-left: 25%;
+  }
+}
+
+.omb_login .omb_authTitle {
+  text-align: center;
+  line-height: 300%;
+}
+
+.omb_login .omb_socialButtons a {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `omb_login` should be written in lowercase with hyphens<br>Selector `omb_socialButtons` should be written in lowercase with hyphens